src/communication/serverMessages.ts

```diff
@@ -25,16 +26,18 @@ export interface Data {
 
 export interface CommandData {
     Command: string;
-    JsonMessage: string;
+    Data: any;
```

Comment:
It's confusing. Perhaps we can improve the naming.
And try to avoid any. If Data can be many things, perhaps you can use discriminated unions ([https://www.typescriptlang.org/docs/handbook/advanced-types.html](url))